packages/ui/v2/form/Select.tsx

```diff
@@ -1,111 +1,62 @@
-import { useSelect } from "downshift";
-import { useState } from "react";
+import ReactSelect, { components, GroupBase, InputProps, Props } from "react-select";
 
-import { classNames } from "@calcom/lib";
+import classNames from "@calcom/lib/classNames";
 
-type IItem =
-  | {
-      itemType: "ListItem";
-      title: string;
-      description: string;
-      value: string;
-    }
-  | {
-      itemType: "Title";
-      title: string;
-      value: string;
-    }
-  | {
-      itemType: "NameCard";
-      img: string;
-      value: string;
-      name: string;
-    }
-  | {
-      itemType: "Spacer";
-    };
-interface SelectProps {
-  items: IItem[];
-}
-
-function itemToString(item: IItem | null) {
-  return item && item.itemType !== "Spacer" && item?.value ? item.value : "";
-}
-
-function Select({ items }: SelectProps) {
-  const [selectedItems, setSelectedItems] = useState<IItem[]>([]);
-  const {
-    isOpen,
-    selectedItem,
-    getToggleButtonProps,
-    getLabelProps,
-    getMenuProps,
-    highlightedIndex,
-    getItemProps,
-  } = useSelect<IItem>({
-    items,
-    itemToString,
-    selectedItem: null,
-    onSelectedItemChange: ({ selectedItem }) => {
-      if (!selectedItem) {
-        return;
-      }
+export type SelectProps<
+  Option,
+  IsMulti extends boolean = false,
+  Group extends GroupBase<Option> = GroupBase<Option>
+> = Props<Option, IsMulti, Group>;
 
-      const index = selectedItems.indexOf(selectedItem);
-
-      if (index > 0) {
-        setSelectedItems([...selectedItems.slice(0, index), ...selectedItems.slice(index + 1)]);
-      } else if (index === 0) {
-        setSelectedItems([...selectedItems.slice(1)]);
-      } else {
-        setSelectedItems([...selectedItems, selectedItem]);
-      }
-    },
-  });
-  const buttonText = selectedItems.length ? `${selectedItems.length} elements selected` : "Elements";
+export const InputComponent = <Option, IsMulti extends boolean, Group extends GroupBase<Option>>({
+  inputClassName,
+  ...props
+}: InputProps<Option, IsMulti, Group>) => {
+  return (
+    <components.Input
+      // disables our default form focus hightlight on the react-select input element
+      inputClassName={classNames("focus:ring-0 focus:ring-offset-0", inputClassName)}
+      {...props}
+    />
+  );
+};
 
+function Select<
```

Comment:
Use old style of select - was trying to make this integrate nicely with tailwind but we may aswell just use react-select.